website/docs/docs/deploy/job-notifications.md

```diff
@@ -4,7 +4,7 @@ id: "job-notifications"
 description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
-Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
+Set up notifications in dbt Cloud so you can receive email or Slack alerts when a job run succeeds, encountered warnings, fails, or is canceled. 
```

Comment:
Should this be just "warns"

Reply:
If we want to be fancy here, we could probably also link through to the docs for source freshness checks and for tests. 

E.g. now that we have warn notifications, you should go and set up some more tests/freshness checks!

Reply:
"warns" is "warning" for short, correct? 

if this is the case, unless we're referring to the UI option itself, it's good practice to use the real work vs any colloquialisms because it's more translation friendly (like Philadelphia is preferred over Philly)

Reply:
@reubenmc : i do like having a "Warns" description early on so both the short form and long form are together. so i chunked this content a little differently by using a bulleted list instead of paragraph style. lemme know what you think!

Reply:
Yup. I like that! Although while I'm being pedantic, this current says "succeeds, encountered warnings, fails, or is canceled." Should it instead say "encounters warnings" so it's all in the present tense? :)

Reply:
it should be a bullet list now, which uses diff wording. you may need to go into the Files changed tab to see it tho